docs/payments.md

```diff
@@ -2,6 +2,25 @@
 
 Payments app adds support for Respa resources to have paid reservations. Transactions are handled by a third party provider such as Bambora Payform.
 
+## Enabling and configuring Payments
+
+There are a couple of required configuration keys that need to be set in order to enable the payments and the third party provider currently in use:
+
+- `RESPA_PAYMENTS_ENABLED`: Whether app is enabled or not. Boolean `True`/`False`. The default value is `False`.
```

Comment:
I would rather put "Whether payments are enabled or not." or something similar, because technically the app is in use always.

Reply:
Alright, changed that one as suggested